README.md

```diff
@@ -17,12 +17,20 @@ Software documentation is released under the Creative Commons license.
 * The custom [GeekDocs theme](https://geekdocs.de/) provides a bare-bones theme with templates and shortcodes for styling the website and content.
 * Specific .css customizations are added on top of GeekDocs in the /static/custom.css, /data/menu/, and /layouts/ files
 
-# Local Build Process
+# Installation and Download Process
```

Comment:
Mixing gerund and verb phrase headings is not a best practice. Exceptions are sections like Before you Begin or Prerequisites. For parallel construction either make all headings gerunds (with the afore mentioned exceptions) or don't use gerund headings. In this readme I think avoiding gerunds might be a better approach, so I suggest the following change:
```suggestion
# Install Hugo
```